.env.example

```diff
@@ -30,7 +30,7 @@ REDIS_URL=redis://redis:6379/0
 # This environment variable is used to skip user email verification after signup.
 # Set to false for local testing. When set to true, ensure SMTP credentials are
 # configured correctly so that verification emails can be sent to users.
-USER_EMAIL_VERFICATION=false
+USER_EMAIL_VERIFICATION=false
```

Comment:
@AngeloB-AIS can we please also make sure where this env is used, whether right spelling is being used or not?
Or have you already verified that?

Reply:
Hey, just seeing this @sumitd94. The misspelling is only present in this .env.example file. 
<img width="1007" alt="Screenshot 2024-11-05 at 8 46 01 PM" src="https://github.com/user-attachments/assets/1765c2f2-4735-48f1-a8b7-ab666836ed67">
<img width="613" alt="Screenshot 2024-11-05 at 8 45 49 PM" src="https://github.com/user-attachments/assets/7349ca27-66d5-4b25-894a-01303ae979c0">